src/osemgrep/language_server/scan_helpers/Scan_helpers.mli

```diff
@@ -14,6 +14,9 @@ val run_semgrep :
 val scan_workspace : RPC_server.t -> unit
 (** [scan_workspace server] scans the workspace of the given session. *)
 
+val scan_open_documents : RPC_server.t -> unit
```

Comment:
maybe add that it's now the default way to scan and that we don't scan_workspace anymore by default